Src/Support/Google.Apis.Tests/System.Net/AsyncStream.cs

```diff
@@ -99,4 +99,8 @@ public AsyncStream(Stream stream)
     }
 }
 
+#elif NET452 || NET46
+// Nothing required
+#else
+#error Unsupported platform
```

Comment:
I prefer causing a compile error in this situation, so if we add a new target framework to the csproj later then it forces us to consider what the right thing to do is at every bit of framework-specific code.
This means we won't get any errors due to adding a new platform, and not remembering about a bit of conditional code.